lib/sqlalchemy/sql/selectable.py

```diff
@@ -2126,7 +2151,7 @@ def union_all(self, *other: _SelectStatementForCompoundArgument) -> CTE:
             self.element
         ), f"CTE element f{self.element} does not support union_all()"
 
```

Comment:
**mike bayer** (zzzeek) wrote:

remove

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4325